config/routes.rb

```diff
@@ -1,8 +1,12 @@
 Rails.application.routes.draw do
-  get 'likes/index'
-  get 'comments/index'
   # Define your application routes per the DSL in https://guides.rubyonrails.org/routing.html
 
   # Defines the root path route ("/")
-  # root "articles#index"
+
+  root 'users#index'
+
+  get '/users/:user_id/posts' => 'posts#index', as: :user_posts
+  get '/users/:user_id/posts/:id' => 'posts#show', as: :user_post
+  get '/users' => 'users#index'
+  get '/users/:id' => 'users#show'
```

Comment:
- [ ] [Optional] It would be better professionally if you add these routes in this form:
```ruby
resources :users, only: [:index, :show] do
    resources :posts, only: [:index, :show]
end
 ```